### ISSUE TITLE
Warning on 'Sequential evaluation is strongly discouraged. Proceed with caution.'?

### ISSUE THREAD

**Issue by HenrikBengtsson** [nhejazi/biotmle#71]
Just spotted:
```
Warning in biomarkertmle(se = illuminaData[1:2, ], varInt = varInt_index,  :
  Sequential evaluation is strongly discouraged. 
 Proceed with caution.
```
while running rev-dep checks.  What the reason and rational for that warning?  As it's written now, it sounds like something will go wrong or one cannot trust the results.

https://github.com/nhejazi/biotmle/blob/32e00bc69e9d7cfcd738b67a6ababdf2afc7c950/R/biomarkertmle-wrapper.R#L152-L155

**Comment by nhejazi** [comment]
Thanks for pointing this out. Back when I first wrote the warning, I had intended for it to simply communicate that running the procedure sequentially in a loop over many probes would be a very slow process. While that's still the case, I agree that the warning sounds like something much more severe will result from sequential evaluation, which is not the case. I'll correct this in an imminent commit.
